lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -801,13 +806,60 @@
       }
 
       .wrapper-extended {
-        padding: ($baseline/4) 0;
+        display: flex;
+        flex-flow: column wrap;
+
+        .wrapper-icon-message {
+          width: 100%;
+          display: inline-flex;
+          flex-wrap: wrap;
+
+          svg {
+            flex-shrink: 0;
+            width: 43px;
+            height: 43px;
+            margin-left: -5px;
+          }
 
-        .message-copy {
-          display: inline-block;
+          .message-copy {
+            margin-left: $baseline/2;
+            min-width: 15rem;
+            flex: 2;
+
+            .message-copy-bold {
+              font-weight: 600;
+            }
+          }
+
+          a.verified-info {
+            color: #454545;
+            text-decoration: underline;
+          }
+        }
+
+        .action-upgrade-container {
+          flex-shrink: 0;
+          flex-grow: 1;
+          align-self: flex-end;
+        }
+
+        @include media-breakpoint-down(xs) {
+          .action-upgrade-container {
+            width: 100%;
+            margin-top: $baseline;
+          }
+
+          .action-upgrade {
+            width: 100%;
+            flex: 2 100%;
+          }
+        }
+
+        @include media-breakpoint-up(xl) {
+          flex-flow: nowrap;
 
-          .message-copy-bold {
-            font-weight: 600;
+          .action-upgrade-container {
+            margin-left: 16px;
```

Comment:
Can you use `rem` here too?

Reply:
Discussed over slack, this would probably be a weird rem fraction so leaving it as px.